### ISSUE TITLE
Angular animation and loading template not working together

### ISSUE THREAD

**Issue by C0ZEN** [ngneat/transloco#264]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

The loading template does not change his state during a page animation.

## Expected behavior

Once the language/scope is loaded, the loading template should detects the change and update the view.

## Minimal reproduction of the problem with instructions

You can check this [video](https://www.youtube.com/watch?v=jX9nULqMPiQ&feature=youtu.be) showing the problem.  

The "skeleton design" is still visible in the DOM despite that the scope is already loaded.
The content of the `<ng-template #x>` is visible at the same time as the content of `<ng-container *transloco="let translation; scope: 'x'; loadingTpl: x">`  

You can also see on [StackBlitz](https://stackblitz.com/edit/angular-hd4dme?file=src%2Fapp%2Fconstants%2Fanimation-speed.ts) the error.  

In StackBlitz:
- You start from the home page: it's ok (i18n in cache)
- You go to the error page: it's ok
- You go back to the home page: it's ok
- You go once more into the home page: it's broken (the loading state stay until the page animation is finished. The i18n is yet cached and even if not cached, the loaded should be faster than the page animation in this case)

I recorded the bug on [StackBlitz](https://www.youtube.com/watch?v=5VxpSq3ubok&feature=youtu.be).

### Typo

```html
<ng-container *transloco="let translation; scope: 'x'; loadingTpl: x">
  <p class="title">
    {{ translation('x.title') }}
  </p>
</ng-container>
<ng-template #x>
  <p class="title title-fake"></p>
</ng-template>
```

### Animation

```typescript
import {
  animate,
  animateChild,
  group,
  query,
  style
} from '@angular/animations';

export const SLIDE_FROM_RIGHT_ANIMATION = [
  style({
    position: 'relative'
  }),
  query(':enter', [
    style({
      height: '100vh',
      left: 0,
      overflow: 'hidden',
      position: 'fixed',
      top: 0,
      width: '100%'
    })
  ]),
  query(':leave', [
    style({
      height: '100vh',
      left: 0,
      overflow: 'hidden',
      position: 'absolute',
      top: 0,
      width: '100%'
    })
  ]),
  query(':leave', [
    style({
      transform: 'translateX(0%)'
    })
  ]),
  group([
    query(':enter', [
      style({
        'z-index': 0
      })
    ]),
    query(':leave', [
      style({
        'z-index': 1
      }),
      animate('350ms ease-out', style({
        transform: 'translateX(100%)'
      }))
    ])
  ]),
  query(':enter', animateChild())
];
```

## Environment

```
"@angular/core": "9.1.1"
"@angular/animations": "9.1.1"
"@ngneat/transloco": "2.14.0"
```

**Comment by C0ZEN** [comment]
**UPDATE 1:**
Added a repro on StackBlitz.

**Comment by C0ZEN** [comment]
@itayod thank you for your time, I will give it a try right now.
You made my week if this is working.

**Comment by C0ZEN** [comment]
Related to #182 and the hack will not longer be useful.

**Comment by itayod** [comment]
You are welcome @C0ZEN :)
But you still have to wait for a new version with that fix to be released in order to test it.

**Comment by shaharkazaz** [comment]
@C0ZEN  released in v2.17.0 🙂

**Comment by C0ZEN** [comment]
@itayod @shaharkazaz the fix is working fine, thank you.
Smooth and beautiful animations in my apps now 😃
